### ISSUE TITLE
Allow trade and hold a coin

### ISSUE THREAD

**Issue by elvishp2006** [chrisleekr/binance-trading-bot#180]
### Describe the solution you'd like

Allow a config per coin to I set how many units that I wanna hold of this coin, that config will allow e.g.: I wanna hold 50 units of ADA, but keep the bot running for this coin, buying and selling US$ 20 per trade
### Describe alternatives you've considered

Remove the trading for the coins that I wanna hold, or transfer to another wallet.

**Comment by hopkinskong** [comment]
What a coincidence! I just made the same issue for the very same feature!

https://github.com/chrisleekr/binance-trading-bot/issues/181

**Comment by chrisleekr** [comment]
@elvishp2006 another good suggestion.

I will put it in the TODO list later.

**Comment by skinner12** [comment]
Yes, it would be a good feature cause I can't trade some coins with this script that I've bought to hold them in a mid-long term.

**Comment by chrisleekr** [comment]
Closing this as a refactored v1 has been released. It's no longer relevant to the latest code.
